src/Npc.js

```diff
@@ -99,6 +99,27 @@ class Npc extends Character {
       this.addItem(newItem);
     });
 
+    // Hydrate equipment
+    if (this.defaultEquipment.length > 0 && !(this.defaultEquipment instanceof Map)) {
+      const eqDefs = this.defaultEquipment;
+      this.defaultEquipment = new Map();
+      for (const slot in eqDefs) {
+        const itemDef = eqDefs[slot];
+        try {
+          let newItem = state.ItemFactory.create(state.AreaManager.getAreaByReference(itemDef), itemDef);
+          newItem.initializeInventory(itemDef.inventory);
```

Comment:
`itemDef` here is a string so `itemDef.inventory` will always be undefined. You're probably safe to remove this line

Reply:
kk!